theme/sass/components/_footer.scss

```diff
@@ -0,0 +1,113 @@
+/* Footer */
+
+.footer {
+    display: flex;
+    justify-content: center;
+    background-color: $accent-color-two;
+
+    .footer-container {
+        display: flex;
+        margin: 0;
+        width: $gutter-size;
+        align-items: flex-start;
+
+        .copyright-statement {
+            flex: 0 0 20%;
+            max-width: 20%;
+            color: white;
+            font-size: 12px;
+        }
+
+        .footer-navlinks,
+        .footer-info-links {
+            display: flex;
+            flex: 0 0 25%;
+            max-width: 25%;
+            justify-content: center;
+            font-size: 14px;
+            padding: 0;
+            color: white;
+
+            &:hover {
```

Comment:
Hover should be insider anchor tag not their container.

Reply:
done